java/core/src/main/java/co/worklytics/psoxy/gateway/impl/ApiDataRequestHandler.java

```diff
@@ -266,20 +311,25 @@ public HttpEventResponse handle(HttpEventRequest request) {
             // return response
             builder.statusCode(sourceApiResponse.getStatusCode());
 
+            // TODO: in slack-analytics use-case, this is NDJSON
```

Comment:
In addition to that, is a ndjson compressed with `application/gzip`, so we'll need need to uncompressed first